app/connector/aws-ddb/src/main/java/io/syndesis/connector/aws/ddb/customizer/DDBConnectorCustomizerQuery.java

```diff
@@ -15,30 +15,29 @@
  */
 package io.syndesis.connector.aws.ddb.customizer;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 import com.amazonaws.services.dynamodbv2.model.AttributeValue;
 import com.google.common.base.Splitter;
 import io.syndesis.connector.aws.ddb.util.Util;
 import io.syndesis.connector.support.util.ConnectorOptions;
 import org.apache.camel.Exchange;
 import org.apache.camel.component.aws.ddb.DdbConstants;
 import org.apache.camel.component.aws.ddb.DdbOperations;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I think for simple things like this we shouldn't bring in a new dependency like commons-lang. Keep in mind hat everything you add, needs to be supported and maintained, inclduing commons-lang.

Reply:
Removed the dependency. It was a transitioned one so it was included anyway, but you are right, adding here explicitly adds maintenance.